### PR TITLE
Modify size tikv node quantity

### PR DESCRIPTION
**Pull request by joey-yez**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
modify the part of size TiKV node quantity, add tikv storage usage, which is actually used.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/size-your-cluster.md [hunk 1]
```diff
@@ -76,13 +76,15 @@ TiDB Cloud deploys TiKV nodes evenly to all availability zones (at least 3) in t
 >
 > When you scale your TiDB cluster, nodes in the 3 availability zones are increased or decreased at the same time. For how to scale in or scale out a TiDB cluster based on your needs, see [Scale Your TiDB Cluster](/tidb-cloud/scale-tidb-cluster.md).
 
-Minimum number of TiKV nodes: `ceil(compressed size of your data ÷ one TiKV capacity) × the number of replicas`
+Recommanded number of TiKV nodes: `ceil(compressed size of your data ÷ TiKV storage usage ratio ÷ one TiKV capacity) × the number of replicas`
 
-Supposing the size of your MySQL dump files is 5 TB and the TiDB compression ratio is 70%, the storage needed is 3584 GB.
+Supposing the size of your MySQL dump files is 5 TB and the TiDB compression ratio is 40%, the storage needed is 2048 GB.
```
Comment (joey-yez) [inline tidb-cloud/size-your-cluster.md:81]:
```suggestion
Supposing the size of your MySQL dump files in CSV format is 5 TB and the TiDB compression ratio is 40%, the storage needed is 2048 GB.
```

File: tidb-cloud/size-your-cluster.md [hunk 2]
```diff
@@ -76,13 +76,15 @@ TiDB Cloud deploys TiKV nodes evenly to all availability zones (at least 3) in t
 >
 > When you scale your TiDB cluster, nodes in the 3 availability zones are increased or decreased at the same time. For how to scale in or scale out a TiDB cluster based on your needs, see [Scale Your TiDB Cluster](/tidb-cloud/scale-tidb-cluster.md).
 
-Minimum number of TiKV nodes: `ceil(compressed size of your data ÷ one TiKV capacity) × the number of replicas`
+Recommanded number of TiKV nodes: `ceil(compressed size of your data ÷ TiKV storage usage ratio ÷ one TiKV capacity) × the number of replicas`
 
-Supposing the size of your MySQL dump files is 5 TB and the TiDB compression ratio is 70%, the storage needed is 3584 GB.
+Supposing the size of your MySQL dump files in CSV format is 5 TB and the TiDB compression ratio is 40%, the storage needed is 2048 GB.
```
Comment (joey-yez) [inline tidb-cloud/size-your-cluster.md:81]:
```suggestion
Supposing the size of your MySQL dump files is 5 TB and the TiDB compression ratio is 40%, the storage needed is 2048 GB.
```

File: tidb-cloud/size-your-cluster.md [hunk 3]
```diff
@@ -76,13 +76,15 @@ TiDB Cloud deploys TiKV nodes evenly to all availability zones (at least 3) in t
 >
 > When you scale your TiDB cluster, nodes in the 3 availability zones are increased or decreased at the same time. For how to scale in or scale out a TiDB cluster based on your needs, see [Scale Your TiDB Cluster](/tidb-cloud/scale-tidb-cluster.md).
 
-Minimum number of TiKV nodes: `ceil(compressed size of your data ÷ one TiKV capacity) × the number of replicas`
+Recommanded number of TiKV nodes: `ceil(compressed size of your data ÷ TiKV storage usage ratio ÷ one TiKV capacity) × the number of replicas`
```
Comment (qiancai) [inline tidb-cloud/size-your-cluster.md:79]:
```suggestion
Recommended number of TiKV nodes: `ceil(compressed size of your data ÷ TiKV storage usage ratio ÷ one TiKV capacity) × the number of replicas`
```

File: tidb-cloud/size-your-cluster.md [hunk 4]
```diff
@@ -76,13 +76,15 @@ TiDB Cloud deploys TiKV nodes evenly to all availability zones (at least 3) in t
...
 
-Supposing the size of your MySQL dump files is 5 TB and the TiDB compression ratio is 70%, the storage needed is 3584 GB.
+Supposing the size of your MySQL dump files is 5 TB and the TiDB compression ratio is 40%, the storage needed is 2048 GB.
+
+Generally, the usage ratio of tikv storage is not recommended to exceed 80%.
```
Comment (qiancai) [inline tidb-cloud/size-your-cluster.md:83]:
```suggestion
Generally, the usage ratio of TiKV storage is not recommended to exceed 80%.
```

File: tidb-cloud/size-your-cluster.md (hunk 1 above)
Comment (qiancai) [inline tidb-cloud/size-your-cluster.md:81]: Would you please help replace GB to GiB for all GB in this doc?

**Review by ilovesoup — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**